internal/core/ports/wallet.go

```diff
@@ -85,8 +85,8 @@ type TransactionManager interface {
 }
 
 type NotificationManager interface {
-	TxChannel() chan TxNotification
-	UtxoChannel() chan UtxoNotification
+	TxChannel() (chan TxNotification, error)
+	UtxoChannel() (chan UtxoNotification, error)
```

Comment:
Because we need to open streams connection with the gRPC server. (see [here](https://github.com/altafan/tdex-daemon/pull/1/files/0d09142cb2a0ab84003520f5beb52b37067a8acd#diff-4b210a1c4c4e9231a0edabcdad00f73182850660eafb9e109e3b420e8c6a20ffR48)) We may need a way to handle properly this "open streams" errors